### PR TITLE
fix(tests): fix for chrome and firefox, update snapshots (win32, chrome/firefox)

### PR DESCRIPTION
**Pull request by chalapkoStanislav**
Fixed several locators for Firefox, 
reinstated wait for profile picture update, 
updated screenshots.

### CODE REVIEW

File: pages/base-page.js [hunk 1]
```diff
@@ -145,6 +145,7 @@ exports.BasePage = class BasePage {
 
   async waitInfoMessageHidden() {
     await this.infoMessage.waitFor({ state: 'hidden' });
+    await this.page.waitForTimeout(400);
```
Comment (daniel-herrero) [inline pages/base-page.js:148]: Couldn't we replace the [timeout](https://playwright.dev/docs/api/class-page#page-wait-for-timeout) to any other [alternatives](https://stackoverflow.com/questions/46919013/puppeteer-wait-n-seconds-before-continuing-to-the-next-line/73676564#73676564)? 

Just as a tip, could we perhaps to wait for a request to be performed? As I see it in the tests that use this method it's being called to 
https://design.penpot.dev/api/rpc/command/update-profile-photo
https://design.penpot.dev/api/rpc/command/update-team-photo

**Review by daniel-herrero — CHANGES_REQUESTED** [review]
Couldn't we wait for a locator or any other action or a network event to happen?

File: pages/base-page.js (hunk 1 above)
Comment (chalapkoStanislav) [reply pages/base-page.js:148]: The issue is that the API request returns 200 before the image actually changes… So, while everything on the UI indicates that the photo has changed, the animation hasn’t yet completed. I’ll try a few more approaches.

If they don’t work, I might stabilize the test with a condition to 'retry the snapshot until it’s successful,' with a maximum of 4 iterations. This way, we can avoid hardcoding a wait time.

Comment (daniel-herrero) [reply pages/base-page.js:148]: Thanks for the explanations. It all sounds totally understandable.

Comment (daniel-herrero) [reply pages/base-page.js]: @chalapkoStanislav I understand you're still working on those approaches. Am I right?

Comment (chalapkoStanislav) [reply pages/base-page.js]: no, I already requested a recheck =)
please look at the last commit

**Pull request merged**